appeals/web/src/server/lib/mappers/appeal.mapper.js

```diff
@@ -0,0 +1,922 @@
+			}
+		}
+	};
+	/** @type {Instructions} */
+	mappedData.appeal.caseOfficer = {
+		display: {
+			summaryListItem: {
+				key: {
+					text: 'Case Officer'
+				},
+				value: {
+					html:
+						dateToDisplayDate(data.appeal.caseOfficer) || 'No project members have been added yet'
+				},
+				actions: {
+					items: [
+						{
+							text: data.appeal.caseOfficer ? 'Change' : 'Add',
+							href: `${currentRoute}/change/case-officer`
+						}
+					]
+				}
+			}
+		}
+	};
+	/** @type {Instructions} */
+	mappedData.appeal.inspector = {
+		display: {
+			summaryListItem: {
+				key: {
+					text: 'Inspector'
+				},
+				value: {
+					html: dateToDisplayDate(data.appeal.inspector) || 'No project members have been added yet'
+				},
+				actions: {
+					items: [
+						{
+							text: data.appeal.inspector ? 'Change' : 'Add',
+							href: `${currentRoute}/change/inspector`
+						}
+					]
+				}
+			}
+		}
+	};
+	/** @type {Instructions} */
+	mappedData.appeal.appellantCase = {
+		display: {
+			tableItem: [
+				{
+					text: 'Appellant Case'
+				},
+				{
+					text: mapDocumentStatus(data.appeal?.documentationSummary?.appellantCase?.status)
+				},
+				{
+					text: dateToDisplayDate(data.appeal?.documentationSummary?.appellantCase?.dueDate)
+				},
+				{
+					html:
+						data.appeal?.documentationSummary?.appellantCase?.status !== 'not_received'
+							? `<a href="${currentRoute}/appellant-case" class="govuk-link">Review</a>`
+							: ''
+				}
+			]
+		}
+	};
+	/** @type {Instructions} */
+	mappedData.appeal.lpaQuestionnaire = {
+		display: {
+			tableItem: [
+				{
+					text: 'LPA Questionnaire'
+				},
+				{
+					text: mapDocumentStatus(data.appeal?.documentationSummary?.lpaQuestionnaire?.status)
+				},
+				{
+					text: dateToDisplayDate(data.appeal?.documentationSummary?.lpaQuestionnaire?.dueDate)
+				},
+				{
+					html:
+						data.appeal?.documentationSummary?.lpaQuestionnaire?.status !== 'not_received'
+							? `<a href="${currentRoute}/lpa-questionnaire/${data.appeal?.lpaQuestionnaireId}" class="govuk-link">Review</a>`
+							: ''
+				}
+			]
+		}
+	};
+
+	return mappedData;
+}
+
+/**
+ * @param {string | null | undefined} value
+ */
+function nullToEmptyString(value) {
+	if (value !== undefined) {
+		return value ? value : '';
+	} else {
+		return undefined;
+	}
+}
+
+/**
+ * @param {string} answer
+ * @param {string} details
+ */
+const formatAnswerAndDetails = (answer, details) => {
+	return answer === 'Yes'
+		? `${buildHtmSpan(answer)}<br>${buildHtmSpan(details)}`
+		: `${buildHtmSpan(answer)}`;
+};
+
+const formatListOfAppeals = (
+	/** @type {{appealId: Number, appealReference: string}[]} */ listOfAppeals
+) => {
+	if (listOfAppeals.length > 0) {
+		let formattedLinks = ``;
+		for (let i = 0; i < listOfAppeals.length; i++) {
+			const appealReferenceFragments = listOfAppeals[i].appealReference.split('/');
+			const shortAppealReference = appealReferenceFragments?.[appealReferenceFragments.length - 1];
```

Comment:
Perhaps the `appealShortReference` function (`appeals/web/src/server/lib/nunjucks-filters/appeals.js`) could be used here rather than duplicating the functionality?